src/DynamoCoreWpf/UI/GuidedTour/GuidesValidationMethods.cs

```diff
@@ -105,6 +108,16 @@ internal static void ExecuteTermsOfServiceFlow(Step stepInfo, StepUIAutomation u
             }
         }
 
+        private static void CloseGuidedTour(object sender, EventArgs e)
```

Comment:
Can we name it more specific related to PMTou? e.g. `OnPackageManagerTouClosed()`

Reply:
As the event is added to a local object, it will be disposed when the window is closed. What do you think?